### PR TITLE
remove space after para with footnote reference

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/cnx-recipes/issues/3315

### CODE REVIEW

**Comment by wilkus27** [comment]
This is a huge change :thinking: It might cause some regressions. I'm not sure what would be the best way to test it. Please, let me know if you have any ideas :slightly_smiling_face:

**Comment by lwtchu** [comment]
@wilkus27 i think we should test in the original affected book, one footnote from each theme, and one footnote in the preface, table, regular para, and appendix? what do you think?

**Comment by wilkus27** [comment]
For footnotes I think it's ok. We should cover most of the cases with these tests. :relaxed: 

But within this PR I also moved import of `reset` file above the `common-styles` file, because otherwise styles from `reset` overwrite those from the second file:

image

It's hard to predict what regression this change could cause :crying_cat_face:

**Comment by lwtchu** [comment]
@wilkus27 hmmm okay - i say for the english openstax books at least, we will regression test before we release and once we get a QA person in we can coordinate a bigger regression testing cycle. do you think the polish books could wait? or do you want to schedule in a round of testing for them?

File: recipes/mixins/_modify.scss [hunk 1]
```diff
@@ -1251,3 +1251,16 @@
     }
   }
 }
+
+// tag paragraphs that contain footnotes references
```
Comment (LostMojo) [inline recipes/mixins/_modify.scss:1255]: Thank you for leaving notes, this would be very hard to sort through otherwise

**Review by LostMojo — APPROVED** [review]

File: recipes/mixins/_modify.scss (hunk 1 above)
Comment (LostMojo) [reply recipes/mixins/_modify.scss:1255]: Approved, given that tests work out okay.

**Review by Stackblocks — APPROVED** [review]

**Pull request merged**